ts/src/config.ts

```diff
@@ -149,6 +153,8 @@ export interface ProfilerConfig extends GoogleAuthOptions {
   localProfilingPeriodMillis: number;
   localLogPeriodMillis: number;
   localTimeDurationMillis: number;
+  sourcemapPaths?: string[];
+  sourcemap?: SourceMapper;
```

Comment:
Why is this object in the config? Previously the config only contained plain data. This is the first object with behaviours that we are adding.

Reply:
Moved this to the Profiler.
I'd originally put it into the config since the create() method is asynchronous, so I can't call the method from the constructor.  I'm now initializing the source map in the runLoop() function in the profiler.